src/openforms/appointments/tasks.py

```diff
@@ -21,7 +18,7 @@
     ignore_result=False,
     once={"graceful": True},  # do not spam error monitoring
 )
-def maybe_register_appointment(submission_id: int) -> None | str:
+def register_appointment(submission_id: int) -> None | str:
```

Comment:
Shouldn't this task be removed?

Reply:
No, this was discussed with Sergei and we decided that this is the correct way to do it (as a celery task). We just remove the old-flow parts from the task itself.

Reply:
Ahhh okay, check 👍